src/components/common/Extensions/UserNavigation/Token/Token.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { FC } from 'react';
+import TokenIcon from '~shared/TokenIcon/TokenIcon';
```

Comment:
`import TokenIcon from '~shared/TokenIcon'`

Please change it on every place in your pull request.

Reply:
Fixed